pages/common/7za.md

```diff
@@ -0,0 +1,19 @@
+# 7za
+
+> A file archiver with highest compression ratio
```

Comment:
"highest" is a bit unclear here -- highest than anything else currently available? highest possible? I would suggest replacing with just "high".